recipes/nutch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nutch" %}
+{% set version = "1.10.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "17cf195db88b11c5d283448fe362d483e78db58dc564339583de629790103782" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
Might want to do a `python-` prefix here as this is a Python client to Apache Nutch. Guessing we want our own copy of Apache Nutch down the road.